suite-common/wallet-graph/src/graphThunks.ts

```diff
@@ -64,6 +66,7 @@ const getSingleAccountTimeFrameItems = async (
     fiatCurrency: FiatCurrencyCode,
     timeFrame: LineGraphTimeFrameValues,
 ) => {
+    let zeroFiatRatesFallback = null;
```

Comment:
I think this should be typed. If we assign anything to this var, it will use type inference because we're not saying what type it is.

Reply:
Overtyping is anti pattern, it doesn't help type safety and then it makes problem during refactoring. You should let inheritance as much work as possible.

In modern type langues you are defining very few types only and rest is inherited